internal/subscriber/utils.go

```diff
@@ -168,6 +171,17 @@ func (s *Subscriber) isWatchedMsgType(msgType string) bool {
 	return ex
 }
 
+// isWatchedQueryID returns true if the given query ID was added to the subscriber's watched
+// Query IDs list.
+func (s *Subscriber) isWatchedQueryID(queryID uint64) bool {
+	// Empty list represents that all query IDs will be processed.
+	if len(s.watchedQueryIDs) == 0 {
+		return true
+	}
+	_, ex := s.watchedQueryIDs[queryID]
+	return ex
+}
+
```

Comment:
there's a [Registry](https://github.com/stafiprotocol/neutron-query-relayer/blob/26395bb01d3b6ae6fba52a917734a8077fbab9ad/internal/registry/registry.go#L20-L24) struct that tracks addresses that the relayer handles ICQs of. I suppose that the `Registry` is the right place to keep the code related to this part of the relayer's configuration. please refer to the way it handles [Addresses](https://github.com/stafiprotocol/neutron-query-relayer/blob/26395bb01d3b6ae6fba52a917734a8077fbab9ad/internal/registry/registry.go#L5) configuration and make kind of the same for query IDs

Reply:
Good suggestion 👍